<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
    <string notr="true">background-color: rgb(106, 106, 255);</string>
   </property>
    <property name="text">
     <string>YES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 28);</string>
   </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 55, 55);</string>
   </property>
    <property name="text">
     <string>NO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 130, 130);</string>
   </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
